src/element/types.ts

```diff
@@ -116,6 +116,14 @@ export type NonDeleted<TElement extends ExcalidrawElement> = TElement & {
 
 export type NonDeletedExcalidrawElement = NonDeleted<ExcalidrawElement>;
 
+/**
+ * Indicates the color of each character at the position of the key in the record.
+ * If a key doesn't exist then the character is assumed to be the standard text color.
```

Comment:
I found a possible issue: In the Vercel preview, I created a new text element with one sentence of text.  I changed the text color of one word, but then I noticed the JSON object had a color defined for every character in the text element.  Is that the intended behavior?  If not, I'm not sure offhand where the issue lies.  But my guess would be in `src/actions/actionProperties.tsx` or in `src/utils.ts`.

Reply:
Yep that's expected with the current implementation, the structure is essentially a sparse array.